### PR TITLE
Update integrate.quad documentation

### PR DESCRIPTION
**Pull request by caos21**
DOC: Updated epsrel and epsabs documentation entries.

#### Reference issue
#11450

#### What does this implement/fix?
Extended documentation for epsrel and epsabs

#### Additional information
Minor correction in an error message: errabs changed to epsabs

### CODE REVIEW

File: scipy/integrate/quadpack.py [hunk 1]
```diff
@@ -102,9 +102,14 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     Other Parameters
     ----------------
     epsabs : float or int, optional
-        Absolute error tolerance.
+        Absolute error tolerance. Default is 1.49e-8. Quad tries to obtain
+        an accuracy of `abs(i-result) <= max(epsabs, epsrel*abs(i))`
```
Comment (rgommers) [inline scipy/integrate/quadpack.py]: A note on reST formatting: typesetting as code requires double backticks (single backticks means "create a link to this function/object").

File: scipy/integrate/quadpack.py [hunk 2]
```diff
@@ -102,9 +102,14 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
...
+        where `i` = integral of `func` from `a` to `b`, and `result` is the
```
Comment (rgommers) [inline scipy/integrate/quadpack.py]: same here, `i` is code (a/b/func are parameters, there single backticks are fine)

File: scipy/integrate/quadpack.py [hunk 3]
```diff
@@ -102,9 +102,14 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
...
+        numerical approximation. See `epsrel` below.
     epsrel : float or int, optional
-        Relative error tolerance.
+        Relative error tolerance. Default is 1.49e-8.
+        If `epsabs <=0`, `epsrel` must be greater than both 5e-29
```
Comment (rgommers) [inline scipy/integrate/quadpack.py]: double backticks for `epsabs <=0` (and space after `=`)

File: scipy/integrate/quadpack.py [hunk 4]
```diff
@@ -102,9 +102,14 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
...
+        and 50*(machine epsilon). See `epsabs` above.
```
Comment (rgommers) [inline scipy/integrate/quadpack.py]: `50*(machine epsilon)` -> double backticks and spaces around `*`

**Review by rgommers — CHANGES_REQUESTED** [review]
Thanks @caos21. I noticed that other functions in this file (e.g. `dblquad`) have the same parameters, so would be good to keep the descriptions in sync.

**Comment by caos21** [comment]
@rgommers Thanks for the feedback. I hope this commit fix the typos. Also, I hinted at the `dblquad` documentation. If it is OK, I can propagate the fix to the similar functions.

**Comment by ev-br** [comment]
Looks like a nice addition, so let's merge it. Further improvements welcome in a next PR. Thanks @caos21, Ralf.

**Comment by mdhaber** [comment]
Thanks @caos21! If you'd like your name added to [THANKS.txt](https://github.com/scipy/scipy/blob/master/THANKS.txt), please make an entry [here](https://github.com/scipy/scipy/wiki/*THANKS.txt-additions-modifications-for-1.5.0*).

**Pull request merged**